pages/api/video.ts

```diff
@@ -81,7 +82,7 @@ async function createVideo(tmpDir: string, input: string, recordingId: string):
       console.log("Error:", err);
       reject(err);
     })
-    .outputOptions(["-c:v libx264", "-r 30", "-pix_fmt yuv420p"])
+    .outputOptions(["-c:v libx264", "-r 30", "-pix_fmt yuv420p", `-vf "scale=100:-2,setdar=16/9"`])
```

Comment:
> If one and only one of the values is -n with n >= 1, the scale filter will use a value that maintains the aspect ratio of the input image, calculated from the other specified dimension. After that it will, however, make sure that the calculated dimension is divisible by n and adjust the value if necessary.

http://ffmpeg.org/ffmpeg-filters.html#scale-1